.github/workflows/push_pr.yml

```
@@ -28,13 +28,8 @@ jobs:
   compile_and_test:
     strategy:
       matrix:
-        flink: [ 1.18-SNAPSHOT ]
-        jdk: [ '8, 11, 17' ]
-        include:
-          - flink: 1.19-SNAPSHOT
-            jdk: '8, 11, 17, 21'
-          - flink: 1.20-SNAPSHOT
-            jdk: '8, 11, 17, 21'
+        flink: [ 2.1-SNAPSHOT ]
+        jdk: [ '11', '17, 21' ]
```

```suggestion
        jdk: [ '11, 17, 21' ]
```
is there a reason to isolate jdk11 in such a way?

---

No, not really, fixing it, thanks!